packages/create-gasket-app/lib/commands/create.js

```diff
@@ -118,7 +118,7 @@ createCommand.action = async function run(appname, options, command) {
 
     const pluginGasket = makeGasket({
       ...context.presetConfig,
-      plugins: context.presets.concat(context.presetConfig.plugins)
+      plugins: context.presetConfig.plugins.concat(context.presets)
```

Comment:
This is just an annoying limitation of TypeScript. Because `context.presets` is of type `Array<Plugin>` it cannot concatenate in `Array<Plugin | { default: Plugin }>` because `concat` is defined such where the return value is supposed to be the same as the original array. However, it can do this in the reverse order. I assume that the order of plugins doesn't matter.